Windows.md

```diff
@@ -67,6 +67,7 @@ After the build directory it will take optional parameters modifying the CMake c
 | /L | Enables Linter |
 | /O | Enables OpenCV |
 | /PDH | Enables Performance Monitor |
+| /P | Enables Prometheus |
```

Comment:
shouldn't this be `PR`?

Reply:
Good catch, fixed in 4f53b2c73daee9f416fab8ad038404064002d0e8